### ISSUE TITLE
The `` doesn

### ISSUE THREAD

**Issue by nyngwang** [mrbjarksen/neo-tree-diagnostics.nvim#14]

**Comment by nyngwang** [comment]
@mrbjarksen Yes, I just tried the mapping `['<cr>'] = 'preview'` and it works like my `no_open` above. Could you elaborate more on how to make this "preview" mode behavior enabled by default, so I don't have to use a mapping to enable it?

**Comment by mrbjarksen** [comment]
Currently, automatically starting preview mode can only be done by adding an event handler for `neo_tree_buffer_enter`. It should be simple to add a config option for this though, I'll work on that when I can tomorrow. There's a newly opened issue on neo-tree regarding this feature (nvim-neo-tree/neo-tree.nvim#545) so the option might change in the future.

By the way, I believe neo-tree has the `toggle_preview` command using a floating window mapped to `P` by default.

**Comment by nyngwang** [comment]
The commit seems to cause some error:

```
Error in event handler for event neo_tree_buffer_enter[diagnostics.neo_tree_buffer_enter]: ...im/site/pack/packer/start/nui.nvim/lua/nui/tree/init.lua:254: Expected Lua number
```
